features/isolcpus/conf_isolcpus.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
+
+### This script updates the cluster nodes to configure the kernel isolcpus
+### The first part of the script updates the nodes with a custom build hyperkube and
+### the cluster with a custom built MCO image.
+### The script then creates a node configuration for isolcpus and reservedSystemCPUs
+
+usage(){
+	echo "usage::conf_isolcpus -i [isolcpus] -r [reserved_cpus]"
+	echo "-i isolcpus: specify the isolcpus kernel option to be set on the nodes"
+	echo "-r reserved cpus:  explicitly define the cpu list that will be reserved for system on each node"
+	echo "For example: on a 24 CPUs system reserved-cpus=0,1,2,3, then cpu 0,1,2,3 will be reserved for the system"
+	echo "and isolcpus=4-23 will be isolated for containers"
+}
+
+while getopts i:r:h option
+do
+case "${option}"
+in
+i) isolcpus=${OPTARG};;
+r) reservedCPU=${OPTARG};;
+h) usage; exit 0;;
+\?) usage; exit 1;;
+esac
+done
+
+if ([ -z "$isolcpus" ] || [ -z "$reservedCPU" ]) then
+ usage
+ exit 1
+fi
+
+pathToHyperkube="https://github.com/vladikr/hyperkube/raw/master/hyperkube"
```

Comment:
I've used git-lfs to keep this file, but this was a mistake as there is a quota on bandwidth...
I need to find another place to keep it public.

Reply:
This workaround can be now removed I believe. The backport to openshift 4.3 was approved: https://github.com/openshift/origin/pull/24224

Reply:
Yeah,just verified with latest ocp 4.3 build - we can safely remove this part

Reply:
I think we can abandon this PR, because we already have updated kubelet under the OCP 4.3 and you can set `RESERVED_CPUS` and `ISOLATED_CPUS` via manifests

Reply:
> I think we can abandon this PR, because we already have updated kubelet under the OCP 4.3 and you can set `RESERVED_CPUS` and `ISOLATED_CPUS` via manifest

Agree. But, let's get #36 merged first